### DISCUSSION TITLE
[react-pdf] React 18 upgrade 'Page' cannot be used as a JSX component.

### DISCUSSION THREAD

**Discussion by JovanaVar** [DefinitelyTyped/DefinitelyTyped#64569] [Issues with a @types package]
Upgrade to react 18 @types/react-pdf breaks with 
'Document' cannot be used as a JSX component.
Its instance type 'Document' is not a valid JSX element.
 The types returned by 'render()' are incompatible between these types.
Type 'React.ReactNode' is not assignable to type 'import("/node_modules/@types/react/index").ReactNode'.

Same for Page

[a comment by an automated account is not shown]

**Comment by nhardy** [comment]
@JovanaVar `@types/react-pdf` specifies `@types/react@*` in its dependencies. Is it possible that you have multiple incompatible versions of the `@types/react` package installed locally in your project?

Can you run `npm ls @types/react` (or equivalent) to show which versions of the React types you have installed? You may need to ensure that you only have a single version.

**Reply by JovanaVar** [reply]
Thanks❤️
